src/ur/utils/visualization/graph_test.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/rocketlaunchr/dataframe-go"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/test"
+)
```

Comment:
Just one note on your tests - in the future please also test the unhappy path.

Test erroneous inputs, malformed path names and bad data passed in.

For example - if your trajectory planner ever produces in `NaN` or infinite joint positions, we will panic in your math function.

There is an easy fix, I believe I saw  an `NaN` sanitizer in rocket launch's df API